adr/5_escaping_of_column_names_and__as_seperator.md

```diff
@@ -0,0 +1,132 @@
+# Proposal: Use Double Colon (`__`) as a Separator for Nested Fields Representation
+
+- Quesma's generated table or column names should be lowercased alphanumerical character and `_`.
+  - All names must start with character or `_`.
```

Comment:
If you start with `_`, then it gets tricky with flattening nested structures. What's the name of the  nested property in `host___name`? is it `host._name` or `host_.name`?